test/unittest/dds/participant/ParticipantTests.cpp

```diff
@@ -2927,13 +2927,6 @@ TEST(ParticipantTests, CreateTopicNegativeClauses)
     topic = participant->create_topic("footopic", "fake_type_name", TOPIC_QOS_DEFAULT);
     ASSERT_EQ(topic, nullptr);
 
-    // Check that the topic is not created if a non supported durability QoS is provided
-    TopicQos tqos;
-    participant->get_default_topic_qos(tqos);
-    tqos.durability().kind = PERSISTENT_DURABILITY_QOS;
-    topic = participant->create_topic("footopic", type.get_type_name(), tqos);
-    ASSERT_EQ(topic, nullptr);
-
```

Comment:
Should we add a positive case with this?

Reply:
I think we should